docs/providers/okta.md

```diff
@@ -0,0 +1,174 @@
+```bash
+# Base URL construction:
+# https://${OKTA_DOMAIN}.${OKTA_URL}.com/oauth2/${OKTA_SERVER_ID}/v1
+
+
+#######################################
+#
+# MANDATORY nuxt.config.js variables
+#
+
+# OKTA_DOMAIN
+# the unique portion of your okta url, e.g.
+# https://dev-012345-admin.okta.com
+# nuxt.config.js - apply to `auth.strategies.okta.domain`
+OKTA_DOMAIN='dev-012345'
+
+# OKTA_CLIENT_ID
+# the Client ID set by Okta
+# Can be located under:
+# Applications >> <app> >> General >> Client Credentials
+# nuxt.config.js - apply to `auth.strategies.okta.client_id`
+OKTA_CLIENT_ID='0abcde1fgHIjk2lMn345'
+
+
+#######################################
+#
+# OPTIONAL nuxt.config.js variables
+# values shown below are the default values
+#
+
+# OKTA_URL
+# the base okta url
+OKTA_URL='okta.com'
+
+# OKTA_REDIRECT_CALLBACK_URI
+# is the uri Okta will redirect the user to upon login
+# _MUST_ also be set in Okta's
+# Applications >> <app> >> General >> Login redirect URIs
+# default value is '/implicit/callback'
+# nuxt.config.js: apply to `auth.redirect.callback`
+REDIRECT_CALLBACK_URI='/implicit/callback'
+
+# OKTA_REDIRECT_LOGOUT_URI
+# is the uri Okta will redirect the user to upon logout
+# _MUST_ also be set in Okta's 
+# Applications >> <app> >> General >> Logout redirect URIs
+# default value is '/'
+# nuxt.config.js - apply to `auth.redirect.logout`
+REDIRECT_LOGOUT_URI='/'
+
+# OKTA_SERVER_ID
+# the generated portion of the url between `oauth2/` and `/v1`.
+# e.g. `abcdef0ghi1JklMNO234` is the server id for https://dev-012345-admin.okta.com/oauth/abcdef0ghi1JklMNO234/v1/authorize?...
+# Can be located under:
+# API >> Authorization Servers
+# default value is 'default'
+# nuxt.config.js - apply to `auth.strategies.okta.server_id`
+OKTA_SERVER_ID='default'
+
+# OKTA_SCOPE
+# Oauth2 scopes that are supported by the app
+# _MUST_ also be included in Okta's 
+# Applications >> <app> >> General >> Scopes
+# default value is 'openid profile email'
+# nuxt.config.js - apply to `auth.strategies.okta.scope`
+OKTA_SCOPE='openid profile email'
+```
+
+### 3. Add the following to the `nuxt.config.js` file
+```js
+
+require('dotenv').config();
+
+export default {
+  modules: [
+    '@nuxtjs/axios',
+    '@nuxtjs/auth',
+    ['@nuxtjs/dotenv', { /* https://github.com/nuxt-community/dotenv-module#options */ }]
+  ],
+
+  auth: {
+    redirect: {
+      callback: process.env.REDIRECT_CALLBACK_URI, // optional
+      logout: process.env.REDIRECT_LOGOUT_URI // optional
+    },
+    strategies: {
+      okta: {
+        url: process.env.OKTA_URL, // optional
+        domain: process.env.OKTA_DOMAIN,
+        client_id: process.env.OKTA_CLIENT_ID,
+        server_id: process.env.OKTA_SERVER_ID, // optional
+        scope: process.env.OKTA_SCOPE, // optional
+      }
+    }
+  }
+}
+```
+
+## 4. Initialize your vuex store
+
+Create an `index.js` file in your `store` directory.  An empty `index.js` is perfectly acceptable for this purpose.
```

Comment:
It's a part of module installement not provider. Can be simply a note about ensure having store dir :)